data/strings/strings.txt

```diff
@@ -28476,42 +28476,42 @@
     tags = android
     en = Volume
     af = Volume
-    ar = مستوى الصوت
+    ar = مقدار
+    az = Həcmi
     be = Гучнасць
-    bg = Сила на звука
+    bg = Обем
```

Comment:
I found better translations directly in Android SDK, can you please add them here?

```
➜  android-34 pwd                     
/Users/alex/Developer/android-sdk/platforms/android-34
➜  android-34 rg -w '"volume_unknown"'
data/res/values-te/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"వాల్యూమ్"</string>

data/res/values-tl/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Lakas ng Tunog"</string>

data/res/values-ru/strings.xml
1290:    <string name="volume_unknown" msgid="4041914008166576293">"Громкость"</string>

data/res/values-ro/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"Volum"</string>

data/res/values-zh-rTW/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"音量"</string>

data/res/values-ca/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"Volum"</string>

data/res/values-it/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"Volume"</string>

data/res/values-cs/strings.xml
1290:    <string name="volume_unknown" msgid="4041914008166576293">"Hlasitost"</string>

data/res/values-is/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Hljóðstyrkur"</string>

data/res/values-zh-rCN/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"音量"</string>

data/res/values-zu/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Ivolumu"</string>

data/res/values-in/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Volume"</string>

data/res/values-el/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Ένταση ήχου"</string>

data/res/values-ja/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"音量"</string>

data/res/values-en-rXC/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"‎‏‎‎‎‎‎‏‎‏‏‏‎‎‎‎‎‎‏‎‎‏‎‎‎‎‏‏‏‏‏‏‎‏‏‏‎‎‎‎‎‎‏‎‏‏‏‏‏‎‎‎‎‏‏‎‏‏‎‎‎‎‎‎‎‎‏‏‏‎‏‎‏‏‏‎‎‎‏‏‎‏‏‎‎‎‎‏‎‏‎‎‏‎‏‎Volume‎‏‎‎‏‎"</string>

data/res/values-lv/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"Skaļums"</string>

data/res/values-da/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Lydstyrke"</string>

data/res/values-ky/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Үн көлөмү"</string>

data/res/values-gu/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"વૉલ્યૂમ"</string>

data/res/values-mr/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"व्हॉल्यूम"</string>

data/res/values-kk/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Дыбыс қаттылығы"</string>

data/res/values-en-rCA/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Volume"</string>

data/res/values-mn/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Дууны түвшин"</string>

data/res/values-en-rIN/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Volume"</string>

data/res/values-zh-rHK/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"音量"</string>

data/res/values-km/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"កម្រិត​សំឡេង"</string>

data/res/values-ms/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Kelantangan"</string>

data/res/values-pt-rBR/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"Volume"</string>

data/res/values-hy/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Ձայն"</string>

data/res/values-b+sr+Latn/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"Jačina zvuka"</string>

data/res/values-be/strings.xml
1290:    <string name="volume_unknown" msgid="4041914008166576293">"Гучнасць"</string>

data/res/values-am/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"የድምፅመጠን"</string>

data/res/values-pl/strings.xml
1290:    <string name="volume_unknown" msgid="4041914008166576293">"Głośność"</string>

data/res/values-fr-rCA/strings.xml
1290:    <string name="volume_unknown" msgid="4041914008166576293">"Volume"</string>

data/res/values/strings.xml
3539:    <string name="volume_unknown">Volume</string>

data/res/values-uz/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Tovush balandligi"</string>

data/res/values-sq/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Volumi"</string>

data/res/values-vi/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Âm lượng"</string>

data/res/values-sv/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Volym"</string>

data/res/values-sl/strings.xml
1290:    <string name="volume_unknown" msgid="4041914008166576293">"Glasnost"</string>

data/res/values-si/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"ශබ්දය ත්‍රීවතාවය"</string>

data/res/values-ur/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"والیوم"</string>

data/res/values-sk/strings.xml
1290:    <string name="volume_unknown" msgid="4041914008166576293">"Hlasitosť"</string>

data/res/values-pt-rPT/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"Volume"</string>

data/res/values-sw/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Kiwango"</string>

data/res/values-pa/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"ਵੌਲਿਊਮ"</string>

data/res/values-tr/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Ses"</string>

data/res/values-sr/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"Јачина звука"</string>

data/res/values-uk/strings.xml
1290:    <string name="volume_unknown" msgid="4041914008166576293">"Гучність"</string>

data/res/values-pt/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"Volume"</string>

data/res/values-gl/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Volume"</string>

data/res/values-es-rUS/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"Volumen"</string>

data/res/values-ar/strings.xml
1292:    <string name="volume_unknown" msgid="4041914008166576293">"مستوى الصوت"</string>

data/res/values-my/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"အသံအတိုးအကျယ်"</string>

data/res/values-ta/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"ஒலியளவு"</string>

data/res/values-fa/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"میزان صدا"</string>

data/res/values-or/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"ଭଲ୍ୟୁମ୍"</string>

data/res/values-th/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"ระดับเสียง"</string>

data/res/values-lt/strings.xml
1290:    <string name="volume_unknown" msgid="4041914008166576293">"Garsumas"</string>

data/res/values-iw/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"עוצמת קול"</string>

data/res/values-fi/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Äänenvoimakkuus"</string>

data/res/values-eu/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Bolumena"</string>

data/res/values-en-rGB/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Volume"</string>

data/res/values-lo/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"ລະດັບສຽງ"</string>

data/res/values-bs/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"Jačina zvuka"</string>

data/res/values-kn/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"ವಾಲ್ಯೂಮ್"</string>

data/res/values-en-rAU/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Volume"</string>

data/res/values-fr/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"Volume"</string>

data/res/values-es/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"Volumen"</string>

data/res/values-et/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Helitugevus"</string>

data/res/values-hr/strings.xml
1289:    <string name="volume_unknown" msgid="4041914008166576293">"Glasnoća"</string>

data/res/values-hu/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Hangerő"</string>

data/res/values-nl/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Volume"</string>

data/res/values-ne/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"मात्रा"</string>

data/res/values-bg/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Сила на звука"</string>

data/res/values-bn/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"ভলিউম"</string>

data/res/values-nb/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Volum"</string>

data/res/values-mk/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Јачина на звук"</string>

data/res/values-af/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Volume"</string>

data/res/values-de/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Lautstärke"</string>

data/res/values-ka/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"ხმა"</string>

data/res/values-hi/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"आवाज़"</string>

data/res/values-ml/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"വോളിയം"</string>

data/res/values-ko/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"볼륨"</string>

data/res/values-az/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"Həcm"</string>

data/res/values-as/strings.xml
1288:    <string name="volume_unknown" msgid="4041914008166576293">"ধ্বনি"</string>
➜  android-34 

```

Reply:
Done